### PR TITLE
math-magicians-setup 

### PR DESCRIPTION
**Pull request by Yemisirach**
### Project requirements

- First, make sure you have [Node.js](https://nodejs.org/en/) installed on your machine
- The easiest way to create a React application is using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- The application should run in the browser without errors

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Yemi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure to add a professional ReadMe to this project  via [this link](https://github.com/microverseinc/readme-template), and your project should comply with the following standard(s):
- [ ] Linter checks are not showing on your Pull Request. Kindly ensure that **all** required linters are properly configured with regard to GitHub actions, and **all** checks are passing. Kindly request another code review when this is done.
- [ ] All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the _FAQ_ section is optional.
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [ ] Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+
+## Acknowledgments
+
+- Microverse for amusing learning resources.
+- Microverse code reviewers.
+- You! 😄 yes you... for visiting.
+
+## 📝 License
+
+LICENCED UNDER [MIT License](LICENSE)
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please generate a license for your project the current link is broken.
We recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+- [ ] **[React Multi Pages]**
+- [ ] **[Server Storage]**
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Submit an issue
+
+[Issue page](https://github.com/yemisirach)
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please replace your issue link with the below link
https://github.com/Yemisirach/Math-magicians-setup/issues

File: README.md [hunk 3]
```diff
@@ -1 +1,76 @@
...
+
+- [ ] **[CSS styling]**
+- [ ] **[React Multi Pages]**
+- [ ] **[Server Storage]**
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Submit an issue
```
Comment (CollinsTatang) [inline README.md:64]: Please it will be nice to change the header name to **`Contributing`**

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Yemisirach , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yemisirach** [comment]
- [ ] update readme base on review request

**Comment by prolajumokeoni** [comment]
Hi @Yemisirach, when I click on the MIT license link on your readme it is broken please update it 👍🏿

File: README.md [hunk 4]
```diff
@@ -1 +1,76 @@
...
+
+## Acknowledgments
+
+- Microverse for amusing learning resources.
+- Microverse code reviewers.
+- You! 😄 yes you... for visiting.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]() licensed.
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly add a link to your MIT license page, this was requested by previous reviewer and will make your readme more professional

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Yemisirach 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ Good project setup


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- It would be better to  add the key features section to  your readme, this will make your project to be properly documented 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Yemisirach** [comment]
- [ ] Add MIT license page Link

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Yemisirach,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**